skins/oasis/js/privacyUpdateNotification.js

```diff
@@ -4,8 +4,30 @@ $(function() {
         // display notification
         var notification = new BannerNotification(message).show();
 
+        function getCookieDomain(hostname) {
+            const parts = hostname.split('.');
+            if (parts.length < 2) {
```

Comment:
This won't happen on our domains, plus if it did it your code would set a cookie on `undefined`, which would throw an error. I think this check is needed.